src/main/java/com/brandon3055/draconicevolution/handlers/ModularArmorEventHandler.java

```diff
@@ -60,7 +60,7 @@ public static void init() {
 
         ATTRIBUTE_HANDLER.register(WALK_SPEED_UUID, Attributes.MOVEMENT_SPEED, ModularArmorEventHandler::getWalkSpeedAttribute);
         ATTRIBUTE_HANDLER.register(WALK_SPEED_UUID, Attributes.FLYING_SPEED, ModularArmorEventHandler::getFlightSpeedAttribute);
-        ATTRIBUTE_HANDLER.register(WALK_SPEED_UUID, ForgeMod.STEP_HEIGHT.get(), ModularArmorEventHandler::getStepHeight);
+        ATTRIBUTE_HANDLER.register(WALK_SPEED_UUID, ForgeMod.STEP_HEIGHT_ADDITION.get(), ModularArmorEventHandler::getStepHeight);
```

Comment:
Why did you switch over to the deprecated one? Is your workspace using an outdated NeoForge version?

Reply:
I used whatever was on the buildscript!

Reply:
Yep, after updating the NeoForge version in the buildscript, I can now see that this is deprecated.  Reverting...